content/en/announcements/contribex-survey.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Contributor Experience Survey
+date: 2025-02-17
+expiryDate: 2025-03-17
+---
+
+<i class="fas fa-bullhorn"></i> Help us [improve the contributor experience]
+(5-10 min). Survey closes March 17.
+
+[improve the contributor experience]:
+  https://opentelemetry.io/blog/2025/contribex-survey
```

Comment:
Intra-site links must be paths, not external URLs (that's why there's [a GH action failure](https://github.com/open-telemetry/opentelemetry.io/actions/runs/13328321390/job/37226692683?pr=6120)):

```suggestion
[improve the contributor experience]: /blog/2025/contribex-survey/
```